.github/workflows/spelling3.yml

```diff
@@ -45,12 +68,13 @@ on:
   pull_request_target:
     branches:
     - "**"
-    tags-ignore:
-    - "**"
     types:
     - 'opened'
     - 'reopened'
     - 'synchronize'
+  issue_comment:
+    types:
+    - 'created'
```

Comment:
What does this do? Check spelling when someone creates an issue?

Reply:
When someone comments on an issue/PR it will trigger the workflow. Each job independently decides whether to do anything.

For,
https://github.com/microsoft/winget-cli/blob/64d4149c1cf690f66c6a01cfaa9d1eb2fba18ac3/.github/workflows/spelling3.yml#L80
It'll do nothing:
https://github.com/microsoft/winget-cli/blob/64d4149c1cf690f66c6a01cfaa9d1eb2fba18ac3/.github/workflows/spelling3.yml#L90

For,
https://github.com/microsoft/winget-cli/blob/64d4149c1cf690f66c6a01cfaa9d1eb2fba18ac3/.github/workflows/spelling3.yml#L121
It'll do nothing:
https://github.com/microsoft/winget-cli/blob/64d4149c1cf690f66c6a01cfaa9d1eb2fba18ac3/.github/workflows/spelling3.yml#L128

For,
https://github.com/microsoft/winget-cli/blob/64d4149c1cf690f66c6a01cfaa9d1eb2fba18ac3/.github/workflows/spelling3.yml#L136
It'll do nothing:
https://github.com/microsoft/winget-cli/blob/64d4149c1cf690f66c6a01cfaa9d1eb2fba18ac3/.github/workflows/spelling3.yml#L144

It will only do work here:
https://github.com/microsoft/winget-cli/blob/64d4149c1cf690f66c6a01cfaa9d1eb2fba18ac3/.github/workflows/spelling3.yml#L153-L165
One of the criteria being that it isn't this repository.

If someone creates a PR within a single fork and it triggers a comment w/ a note about items that should be added to expect, the comment will include something like this from @check-spelling-bot:
![](https://raw.githubusercontent.com/check-spelling-bot/videos/quote-code/quotecode.gif)

Reply:
I'm dropping this for now.